src/sourmash/cli/tax/label.py

```diff
@@ -0,0 +1,51 @@
+"""add taxonomy information to gather results"""
+
+import sourmash
+from sourmash.logging import notify, print_results, error
+
+
+def subparser(subparsers):
+    subparser = subparsers.add_parser('label')
+    subparser.add_argument(
+        '-g', '--gather-csv', nargs='*', default = [],
+        help='csvs from sourmash gather'
```

Comment:
```suggestion
        help='CSV output files from sourmash gather'
```
or something similar